vars/processAndRunOsdIntegTest.groovy

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Not understanding why we need to add this library separately? What is the difference?

Reply:
It moves logic here https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch-dashboards/integ-test.jenkinsfile#L251-L290 to avoid duplicating the same code in if-else block.